<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RocketAssetSelectionWidget</class>
 <widget class="QWidget" name="RocketAssetSelectionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>292</width>
    <height>23</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#RocketAssetSelectionWidget {
	background-color: red;

}

QLineEdit#assetRefLineEdit {
	font-family: &quot;Arial&quot;;
	font-size: 12px;
	border: 1px solid rgb(154, 154, 154);
	border-radius: 4px; 
	border-top-right-radius: 0px;
	border-bottom-right-radius: 0px;
	border-right: 0px;
	padding-left: 2px;
	padding-right: 2px;
}

QPushButton#buttonStorageSelection, #buttonOpenEditor, #buttonReset { 
	border: 1px solid rgb(154, 154, 154);
	border-radius: 4px; 
	border-top-left-radius: 0px;
	border-bottom-left-radius: 0px;
	background-color: rgb(245,245,245);
	max-width: 26px;
	min-width: 26px;
	max-height: 21px;
	min-height: 21px;
}

QPushButton#buttonOpenEditor, #buttonReset {
	border-radius: 0px;
	border-right: 1px solid transparent;
}

QPushButton#buttonStorageSelection:hover, #buttonOpenEditor:hover, #buttonReset:hover { 
	border: 1px solid rgb(243, 154, 41); 
}

QPushButton#buttonStorageSelection:pressed, #buttonOpenEditor:pressed, #buttonReset:pressed { 
	background-color: rgb(255,255,255);
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLineEdit" name="assetRefLineEdit">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>23</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Texture Reference</string>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="buttonReset">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>28</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>28</width>
       <height>23</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Clear...</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="RocketPlugin.qrc">
       <normaloff>:/images/icon-x-24x24.png</normaloff>:/images/icon-x-24x24.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>12</width>
       <height>12</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="buttonOpenEditor">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>28</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>28</width>
       <height>23</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Open in editor...</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="RocketPlugin.qrc">
       <normaloff>:/images/icon-visual-editor-32x32.png</normaloff>:/images/icon-visual-editor-32x32.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="buttonStorageSelection">
     <property name="minimumSize">
      <size>
       <width>28</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>28</width>
       <height>23</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Pick from storage...</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="RocketPlugin.qrc">
       <normaloff>:/images/icon-download-64x64.png</normaloff>:/images/icon-download-64x64.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>18</width>
       <height>18</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="RocketPlugin.qrc"/>
 </resources>
 <connections/>
</ui>
